tests/tools/common.test.ts

```diff
@@ -49,3 +57,104 @@ describe('isEmptyAddress', function () {
     expect(isEmpty).toBe(false)
   })
 })
+
+describe('isEthAddress', function () {
+  it('0x8ba1f109551bD432803012645Ac136ddd64DBA72', function () {
```

Comment:
The tests are pretty similar.
There should be some comments to help us better understand the purpose of the tests.